liquibase-core/src/main/java/liquibase/database/core/PostgresDatabase.java

```diff
@@ -197,7 +197,7 @@ public boolean generateAutoIncrementBy(BigInteger incrementBy) {
     public String escapeObjectName(String objectName, Class<? extends DatabaseObject> objectType) {
         if (quotingStrategy == ObjectQuotingStrategy.LEGACY && hasMixedCase(objectName)) {
             return "\"" + objectName + "\"";
-        } else if (objectType != null && objectType.isAssignableFrom(LiquibaseColumn.class)) {
+        } else if (objectType != null && objectType.isInstance(LiquibaseColumn.class)) {
             return (objectName != null && !objectName.isEmpty()) ? objectName.trim() : objectName;
         }
```

Comment:
@nvoxland please confirm that this change aligns with was actually intended.
comment states that its "to not quote columns for Liquibase (DBCL/DBCLL) tables", but in fact will be triggered every time when `escapeObjectName("val", Column.class)` will be used for PostgresDB.

I depend on above case, to discriminate between computed and non-computed columns (say `ID` vs `"ID"`).
this logic was not invented, but extended: Oracle treats ordered index column as computed so I just added alike support for MsSQL, Postgres and DB2.
the problem is/will-be when conflict of compute columns starts to happen.
maybe better solution is to convert `"ID"` column name into `"ID" DESC` (plus remove adding DESC part in `CreateIndexSqlGenerator`)
what do you think about it?